doc/reference/reference_lua/box_info/election.rst

```diff
@@ -10,9 +10,25 @@ box.info.election
 
    Since version :doc:`2.6.1 </release/2.6.1>`.
    Show the current state of a replica set node in regards to
-   :ref:`leader election <repl_leader_elect>`, namely,
-   election state (mode), election term, vote in the current term,
-   and the leader ID of the current term.
+   :ref:`leader election <repl_leader_elect>`.
+
+   The following information is provided:
+
+   *    ``state`` -- election state (mode) of the node. Possible values are ``leader``, ``follower``, or ``candidate``.
+        For more details, refer to description of the :ref:`leader election process <repl_leader_elect_process>`.
+        When election is enabled, the node is writable only in the ``leader`` state.
+
+   *    ``term`` -- current election term.
+
+   *    ``vote`` -- ID of a node the current node votes for. If the value is ``0``, it means the node doesn't vote in the current term.
```

Comment:
If the value is `0`, it means the node hasn't yet voted in the current term.

Reply:
Corrected.